public/app/features/canvas/runtime/SceneTransformWrapper.tsx

```diff
@@ -31,7 +45,11 @@ export const SceneTransformWrapper = ({ scene, children: sceneDiv }: SceneTransf
       disabled={!config.featureToggles.canvasPanelPanZoom || !scene.shouldPanZoom}
       panning={{ allowLeftClickPan: false }}
     >
-      <TransformComponent>{sceneDiv}</TransformComponent>
+      <TransformComponent>
+        {/* The <div> element has child elements that allow for mouse events, so we need to disable the linter rule */}
+        {/* eslint-disable-next-line jsx-a11y/no-static-element-interactions */}
+        <div onMouseDown={onSceneContainerMouseDown}>{sceneDiv}</div>
```

Comment:
I'm not sure if adding this wrapper div around sceneDiv will break stuff (like how we calculate things) - testing locally seems ok but want some more 👀 on this